doc/classes/RayCast2D.xml

```diff
@@ -137,13 +138,13 @@
 			If [code]true[/code], collisions will be reported.
 		</member>
 		<member name="exclude_parent" type="bool" setter="set_exclude_parent_body" getter="get_exclude_parent_body" default="true">
-			If [code]true[/code], the parent node will be excluded from collision detection.
+			If [code]true[/code], the parent node will not report collisions with this raycast. See also [method Node.get_parent] and [method add_exception].
```

Comment:
```suggestion
			If [code]true[/code], this raycast will not report collisions with its parent node. This property only has an effect if the parent node is a [CollisionObject2D]. See also [method Node.get_parent] and [method add_exception].
```

Reply:
Mentioning the class directly defeats the purpose of this PR, but it is a good remark nonetheless.